### ISSUE TITLE
Keybind for `Editor::rewrap` prevents `@` from being typed in editor for certain keyboard-layouts on Mac

### ISSUE THREAD

**Issue by MrSubidubi** [zed-industries/zed#18104]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Due to `option-q` being mapped to `@` on certain keyboard-layouts for Mac, I am unable to enter the `@`-symbol in the editor as this conflicts with the keybinding introduced for `Editor::rewrap` in #17953 . 

The affected keyboard-layouts include at least the QWERTZ - and the hungarian layout.

I had to manually remove the keybinding for `Editor::rewrap` in my `keymap.json` with

```json
{
    "bindings": {
        "alt-q": null
    }
},
...
```

While this seems to be the most reasonable keybinding as per https://github.com/zed-industries/zed/issues/4588#issuecomment-2230128399 , I do not think this should cause symbol inputs to not be registered and require a manual unbind.

I believe this might be a Mac-only issue, as non-mac keyboards bind `@` to `Alt Gr + q`, however I am not sure whether GPUI differentiates `Alt` and `Alt Gr`.
 

### Environment

Zed: v0.154.0 (Zed Preview)
OS: macOS 14.6.1
Memory: 8 GiB
Architecture: aarch64

**Comment by miltonparedes** [comment]
Same here 😕, had to fix it with:
```
"ctrl-q": "editor::Rewrap",
"alt-q": null
```

**Comment by MrSubidubi** [comment]
Closed via #20378
